Makefile

```diff
@@ -80,9 +82,9 @@ copy-helpers:
 
 grpc-mock:
 	printf $(COLOR) "Generate gRPC mocks..."
-	mockgen -package operatorservicemock -source operatorservice/v1/service_grpc.pb.go -destination operatorservicemock/v1/service_grpc.pb.mock.go
-	mockgen -package workflowservicemock -source workflowservice/v1/service_grpc.pb.go -destination workflowservicemock/v1/service_grpc.pb.mock.go
-	mockgen -package cloudservicemock -source cloud/cloudservice/v1/service_grpc.pb.go -destination cloud/cloudservicemock/v1/service_grpc.pb.mock.go
+	$(GOBIN)/mockgen -package operatorservicemock -source operatorservice/v1/service_grpc.pb.go -destination operatorservicemock/v1/service_grpc.pb.mock.go
```

Comment:
I don't like this. We don't have in other `Makefile`s and I would like them to be more or less consistent across repos. [Makefile](https://github.com/temporalio/temporal/blob/main/Makefile) in server repo uses slightly different approach. I am ok, if you copy it from there.